src/views/CreateView.vue

```diff
@@ -343,6 +356,23 @@ export default {
         this.isFormSubmitted = true
       }
     }
+  },
+  async mounted() {
+    const fetchExercises = async () => {
+      const response = await supabase.from('Exercises').select()
+      
+      if (response.error === null) {
```

Comment:
Are you sure that response error should be exactly null? I wonder if `responses.ok` isn't a better way

Reply:
I actually explained it here:
> Also, I couldn't use response.ok because this response object didn't have such a property. So I changed it for response.error === null. Maybe I could do just !response.error but I seem to no longer know when I can use this syntax.

I tried this `response.ok` at first, as you suggested it, but  my function didn't work so I logged `response.ok` and found out it returns `undefined`. So, I then logged just `response` and got this:
![response](https://user-images.githubusercontent.com/113309294/222921002-45188023-adb6-4433-a80a-8ce3541809c5.PNG)

Reply:
So you can check `status > 199 && status < 300`